src/screens/dashboard/dashboard.tsx

```diff
@@ -63,12 +71,19 @@ export interface IReduxProps {
     selectedBlockchainAccounts: IAccountState[];
     userCurrency: string;
     chainId: ChainIdType;
+    setNotifications: typeof setNotifications;
+    deviceId: string;
+    fetchNotifications: (page?: number) => Promise<any>;
```

Comment:
Too many actions specific for notifications on this screen that are not actually used

create just one action that handles notifications (register, fetch, getUnseen) and here just show the info from redux (red bullet on the notifications icon)